frontend/appflowy_flutter/lib/core/helpers/url_launcher.dart

```diff
@@ -45,16 +45,18 @@ Future<bool> afLaunchUri(
   }
 
   // try to launch the uri directly
-  bool result;
-  try {
-    result = await launcher.launchUrl(
-      uri,
-      mode: mode,
-      webOnlyWindowName: webOnlyWindowName,
-    );
-  } on PlatformException catch (e) {
-    Log.error('Failed to open uri: $e');
-    return false;
+  bool result = await launcher.canLaunchUrl(uri);
```

Comment:
I remember that `canLaunchUrl` has a side effect: some urls can actually be opened, but this function returns false.

Reply:
There's an issue that opening links like `xxx.com` will show an error dialog. 
This happens because the `result` is false, and the code will try again after adding "http". 
This change is to fix this problem.